posthog/api/test/test_insight.py

```diff
@@ -1939,118 +1928,6 @@ def test_insight_trends_breakdown_pagination(self) -> None:
         self.assertEqual(response.status_code, status.HTTP_200_OK, response.json())
         self.assertIn("offset=25", response.json()["next"])
 
-    def test_insight_trends_breakdown_persons_with_histogram(self) -> None:
```

Comment:
I just couldn't get this test to work. During debugging I found that `raw_sessions` isn't populated in tests, but using a different method to create the events or passing in a valid uuid7 for the session id all didn't work. I couldn't find additional steps in other tests.

Running a session histogram breakdown locally works, e.g.
```json
{
  "events": [
    {
      "id": "$pageview",
      "math": "total",
      "type": "events",
      "order": 0
    }
  ],
  "date_from": "-7d",
  "breakdown": "$session_duration",
  "breakdown_type": "session",
  "breakdown_histogram_bin_count": 2
}
```